auction-server/src/auction/service/auction_manager.rs

```diff
@@ -393,67 +387,103 @@ impl AuctionManager<Svm> for Service<Svm> {
             return Err(anyhow::anyhow!("No bids to submit"));
         }
 
-        let mut bid = bids[0].clone();
-        self.add_relayer_signature(&mut bid);
-        match self.send_transaction(&bid.chain_data.transaction).await {
-            Ok(response) => Ok(response),
-            Err(e) => {
-                tracing::error!(error = ?e, "Error while submitting bid");
-                Err(anyhow::anyhow!(e))
+        let send_futures: Vec<_> = bids
+            .into_iter()
+            .map(|mut bid| {
+                self.add_relayer_signature(&mut bid);
+                async move { self.send_transaction(&bid.chain_data.transaction).await }
+            })
+            .collect();
+
+        let results = futures::future::join_all(send_futures).await;
+        let mut result = None;
+        for res in results.into_iter() {
+            match res {
+                Ok(sig) => {
+                    result = Some(Ok(sig));
+                }
+                Err(e) => {
+                    tracing::error!(error = ?e, "Error while submitting bid");
+                    if result.is_none() {
+                        result = Some(Err(anyhow::anyhow!(e)));
+                    }
+                }
             }
         }
+        result.expect("results should not be empty")
     }
 
     async fn get_bid_results(
         &self,
         bids: Vec<entities::Bid<Svm>>,
         bid_status_auction: entities::BidStatusAuction<entities::BidStatusSvm>,
-    ) -> Result<Option<Vec<entities::BidStatusSvm>>> {
+    ) -> Result<Vec<Option<entities::BidStatusSvm>>> {
         if bids.is_empty() {
-            return Ok(Some(vec![]));
-        }
-
-        if bids.len() != 1 {
-            tracing::warn!(bid_status_auction = ?bid_status_auction, bids = ?bids, "multiple bids found for transaction hash");
+            return Ok(vec![]);
         }
 
         //TODO: this can be optimized out if triggered by websocket events
-        let status = self
+        let signatures: Vec<_> = bids
+            .iter()
+            .map(|bid| {
+                *bid.chain_data
+                    .transaction
+                    .signatures
+                    .first()
+                    .expect("Signature array is empty on svm bid tx")
+            })
+            .collect();
+        let statuses: Vec<_> = self
             .config
             .chain_config
             .client
-            .get_signature_status_with_commitment(
-                &bid_status_auction.tx_hash,
-                CommitmentConfig::confirmed(),
-            )
-            .await?;
+            .get_signature_statuses(&signatures)
```

Comment:
Add comment for this: `An array of transaction signatures to confirm, as base-58 encoded strings (up to a maximum of 256)`

Reply:
this is the documentation for this rpc function. Why should we add it here 🤔

Reply:
Comment here?